TrainingGym/README.md

```diff
@@ -36,5 +36,3 @@ Then
 To deactivate your virtual environment:
 
 ``deactivate``
-
-You will probably need to activate and deactivate each time you open/close the project.
```

Comment:
You don't need to activate and deactivate as long as you run with the pycharm interface, but yeah if you want to run it in the terminal you should do that